package.json

```diff
@@ -49,6 +49,7 @@
     "debug": "^2.2.0",
     "es6-promise": "^3.0.2",
     "js-base64": "^2.1.9",
+    "nock": "^8.0.0",
```

Comment:
This is used only in the test files, so it should be a devDependency not a dependency.